samples/packages/pulumi-typescript/tests/test-bootstrap.ts

```diff
@@ -0,0 +1,99 @@
+// this is the code that would be in a project bootstrap file. It pulls the stack together using a config and CDF ochestrator
+import * as cdf from '@openfabr/cdf'
+import { PackageInfraConfig, PackageInfraPlanConstructs, PackagePlanner } from '../src';
+import config from './short-config.json'
+import * as aws from '@pulumi/aws';
+import { InfraPlanConstructs, GeneralConfig, NetworkConfig, ComponentConfig, ServiceConfig, RelationConfig, InfraConfig, Planner, Custom, ResultHandler, Orchestrator, InfraPlanOutputs } from '@openfabr/cdf';
+import { ProjectComponent, ProjectComponentOpts } from '@openfabr/cdf/build/main/lib/bootstrap/pulumi/component';
+import * as pulumi from '@pulumi/pulumi';
+import { ComponentResource, Output } from '@pulumi/pulumi';
+import {error, log} from 'console'
+
+
+class LocalProjectComponent<
+  IPC extends InfraPlanConstructs,
+  GC extends GeneralConfig,
+  NC extends NetworkConfig,
```

Comment:
this keeps it generic and flexible. In case package authors find the need to pass along other outputs.

the 4 contruct specific properties are for convenience and increase usability.